src/SkosGraphConfigurator.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\rdf_skos;
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+
+/**
+ * Service used to set up the SKOS graphs.
+ */
+class SkosGraphConfigurator implements SkosGraphConfiguratorInterface {
+
+  /**
+   * The config factory.
+   *
+   * @var \Drupal\Core\Config\ConfigFactoryInterface
+   */
+  protected $configFactory;
+
+  /**
+   * SkosGraphSetup constructor.
+   *
+   * @param \Drupal\Core\Config\ConfigFactoryInterface $configFactory
+   *   The config factory.
+   */
+  public function __construct(ConfigFactoryInterface $configFactory) {
+    $this->configFactory = $configFactory;
+  }
+
+  /**
+   * Sets up the graphs.
+   *
+   * @param array $graphs
+   *   A list of graph URIs, keyed by graph names.
+   */
+  public function addGraphs(array $graphs = []): void {
```

Comment:
Since we have an interface where the documentation is present, let's just use 
```
/**
   * {@inheritdoc}
   */
```

Reply:
fixed